Tools/machines/crusher-olcf/crusher_warpx.profile.example

```diff
@@ -2,6 +2,7 @@
 #export proj=<yourProject>
 
 # required dependencies
+module load cce/13.0.1
```

Comment:
If this is loaded before `rocm` (which is HIP but also the compiler package of the `PrgEnv-amd`), the same linker error persists.
Must be moved down.